extensions/SemanticMediaWiki/DefaultSettings.php

```diff
@@ -607,7 +607,7 @@
 #
 # @since 2.3
 ##
-$GLOBALS['smwgValueLookupCacheLifetime'] = 60 * 60 * 24 * 7; // a week
+$smwDefaults['smwgValueLookupCacheLifetime'] = 60 * 60 * 24 * 7; // a week
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)